src/api_helpers.jl

```diff
@@ -325,6 +325,30 @@ end
 ### Table Interface
 ###
 
+function h5tb_get_table_info(loc_id, table_name)
+    nfields = Ref{hsize_t}()
+    nrecords = Ref{hsize_t}()
+    h5tb_get_table_info(loc_id, table_name, nfields, nrecords)
+    return nfields[], nrecords[]
+end
+
+function h5tb_get_field_info(loc_id, table_name)
+    nfields, = h5tb_get_table_info(loc_id, table_name)
+    strlen = 64 # string buffer size
+    field_sizes = Vector{Csize_t}(undef,nfields)
+    field_offsets = Vector{Csize_t}(undef,nfields)
+    type_size = Ref{Csize_t}()
+    while true
+        field_names = [Vector{UInt8}(undef,strlen) for _ in 1:nfields]
+        h5tb_get_field_info(loc_id, table_name, field_names, field_sizes, field_offsets, type_size)
+        if any(!in(0x00, buf) for buf in field_names)
```

Comment:
I'm hoping it's safe to assume HDF5 is nullterminating the string. since that's my assumption.

Reply:
I think that is correct.

Reply:
Yea I know think so to, thanks to your reference below  (`strcpy`). Thanks for checking.